examples/flask/lib/charms/data_platform_libs/v0/data_interfaces.py

```diff
@@ -331,14 +331,10 @@ def _on_topic_requested(self, event: TopicRequestedEvent):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 40
+LIBPATCH = 36
```

Comment:
Also looks like the version is being downgraded here

Reply:
Closing the comment, answer in the ingress library comment